core/test/unit/data/schema/fixtures/utils_spec.js

```diff
@@ -152,19 +152,19 @@ describe('Migration Fixture Utils', function () {
             fixtureUtils.addFixturesForRelation(fixtures.relations[0]).then(function (result) {
                 should.exist(result);
                 result.should.be.an.Object();
-                result.should.have.property('expected', 43);
-                result.should.have.property('done', 43);
+                result.should.have.property('expected', 59);
```

Comment:
Are any of these tests actually useful? would we be better off deleting them?

Reply:
Maybe a question for @kirrg001? They don't seem particularly useful to me, any time the schema changes, especially around permissions there's always a ton of other tests to update too.

Reply:
I agree, it's not comfortable to change multiple tests when the permissions are extended. But i would leave this as it for now? Reasons:

- this is a unit test we might want to keep it instead of other tests (e.g. integration tests)
- we want to switch [to use more unit tests](https://github.com/TryGhost/Ghost/issues/9178#issuecomment-340726993) in the near future, we will reconsider all of tests when we start with overhauling the test env (probably something for cleanup weeks)